ReflectorIOS/Views/ArticleList/ArticleCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ArticleCell.swift
+//  ReflectorIOS
+//
+//  Created by Austin Brown on 9/17/20.
+//  Copyright © 2020 Nikhil Menon. All rights reserved.
+//
+
+import SwiftUI
+fileprivate enum CellConstants {
+    static let titleSize: CGFloat = 20
+    static let lineLimit = 3
+    static let detailSize: CGFloat = 15
+}
+struct ArticleCell: View {
+    private(set) var article: Article
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(article.title ?? "")
+                .font(.system(size: CellConstants.titleSize, weight: Font.Weight.bold))
+                .foregroundColor(Color(Theme.reflectorMaroon!))
```

Comment:
Remember in an earlier comment to this Code Review I said to add a `!` to the `Theme.reflectorMaroon`?
Well if you did that, it now means you can remove the `!` declared here on line 22.